packages/server/src/plugin/landingPage/default/index.ts

```diff
@@ -86,7 +86,9 @@ function ApolloServerPluginLandingPageDefault<TContext extends BaseContext>(
     apolloStudioEnv: 'staging' | 'prod' | undefined;
   },
 ): ImplicitlyInstallablePlugin<TContext> {
-  const version = maybeVersion ?? '_latest';
+  const explorerVersion = maybeVersion ?? 'v3';
+  const sandboxVersion = maybeVersion ?? 'v2';
+  const apolloServerLandingPageVersion = maybeVersion ?? '_latest';
```

Comment:
Can we export these defaults and import them in the test? It'll be nice to see test updates tied to the constants updating.

Reply:
oops missed this, doing now